meteor/client/ui/Shelf/AdLibPanel.tsx

```diff
@@ -157,6 +158,16 @@ export function matchFilter(
 		) {
 			return false
 		}
+		// Hide duplicates
+		if (filter.hideDuplicates && uniquenessIds) {
+			const uniquenessId = item.uniquenessId || unprotectString(item._id)
+			console.log(uniquenessId)
```

Comment:
This should be removed.

Reply:
Removed https://github.com/olzzon/tv-automation-server-core/commit/4e31a4cf8a5decd3a9ba27b40f549b6f6ecdda9b